packages/core/src/components/Modal/Modal.tsx

```diff
@@ -50,10 +51,16 @@ const Component: FC<ModalProps> = memo(
             open && isEscPressed && onCloseModal && onCloseModal();
         }, [open, isEscPressed]);
 
+        const tempHandler = () => {
```

Comment:
Could we please remove all the changes in this file ?

Reply:
Reverted